rpcs3/rpcs3qt/emu_settings_type.h

```diff
@@ -246,14 +246,14 @@ inline static const QMap<emu_settings_type, cfg_location> settings_location =
 	{ emu_settings_type::DisableOcclusionQueries,    { "Video", "Disable ZCull Occlusion Queries"}},
 	{ emu_settings_type::DisableVideoOutput,         { "Video", "Disable Video Output"}},
 	{ emu_settings_type::DisableFIFOReordering,      { "Video", "Disable FIFO Reordering"}},
-	{ emu_settings_type::DisableNativefloat16,       { "Video", "Disable native float16 support"}},
 	{ emu_settings_type::Enable3D,                   { "Video", "Enable 3D"}},
 	{ emu_settings_type::StrictTextureFlushing,      { "Video", "Strict Texture Flushing"}},
 	{ emu_settings_type::ForceCPUBlitEmulation,      { "Video", "Force CPU Blit"}},
 	{ emu_settings_type::DisableOnDiskShaderCache,   { "Video", "Disable On-Disk Shader Cache"}},
 	{ emu_settings_type::DisableVulkanMemAllocator,  { "Video", "Disable Vulkan Memory Allocator"}},
 	{ emu_settings_type::ShaderMode,                 { "Video", "Shader Mode"}},
 	{ emu_settings_type::ShaderCompilerNumThreads,   { "Video", "Shader Compiler Threads"}},
+	{ emu_settings_type::ShaderPrecisionQuality,     { "Video", "Shader Precision"}},
```

Comment:
Missing entry for the option in translation file

Reply:
We have a translation file? :laughing:

Reply:
I cannot find any references to a translation file for the other options, hint required.

Reply:
emu_settings::GetLocalizedSetting